python/ray/worker.py

```diff
@@ -1914,6 +1914,13 @@ def remote(*args, **kwargs):
 
   def make_remote_decorator(num_return_vals, num_cpus, num_gpus, func_id=None):
     def remote_decorator(func_or_class):
+      try:
+        # Handle the Numba case: if this is a Numba-compiled function, get the underlying Python function
+        func_or_class = getattr(func_or_class, func_or_class.__numba__)
```

Comment:
Wouldn't this essentially remove the jitting? So we wouldn't get the benefits of numba? Or am I misinterpreting it?